grunt_tasks/webpack.config.js

```diff
@@ -27,8 +27,7 @@ var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var paths = require('./webpack.paths.js');
 // Resolving the Babel presets here is required to support symlinking plugin directories from
 // outside Girder's file tree
-const es2015BabelPreset = require.resolve('babel-preset-es2015');
-const es2016BabelPreset = require.resolve('babel-preset-es2016');
+const babelPresets = require.resolve('babel-preset-env');
```

Comment:
Is this still necessary?  The plugins using babel presets just use the string `env`.

Reply:
We talked in person and reached the conclusion that we don't know if this is still necessary for symlinked plugins.  The candela plugin was changed because as a core plugin, it will never be symlinked.  Babel 7 claims to have fixes for discovering environments through symlinks.  We can revisit this once that comes out.